README.md

```diff
@@ -21,72 +21,71 @@ This package is just a contract, a config file and a `ServiceProvider`.
 As usual, include the `ServiceProvider` in your `app/config/app.php` file like so:
 
 ```php
-    'providers' => array(
+    'providers' => [
         // ...
-        GuiWoda\RouteBinder\RouteBinderServiceProvider::class, // php5.5, nice! ;-)
+        'GuiWoda\RouteBinder\RouteBinderServiceProvider',
         // ...
-    )
+    ]
 ```
 
 Then, publish the package's configuration:
 
 ```bash
-    php artisan config:publish guiwoda/route-binder
+    php artisan vendor:publish
 ```
 
 Afterwards, you'll need to create some classes that implement the `GuiWoda\RouteBinder\RouteBinder` interface.
 Don't panic! You'll see it's a piece of cake:
- 
-```php
-    namespace App\Http\Routes;
-
-    use GuiWoda\RouteBinder\RouteBinder;
-    use Illuminate\Routing\Router;
 
-    class FooRouteBinder implements RouteBinder
+```php
```

Comment:
If this commit is about whitespace changes, please keep them consistent across the whole file.
If it isn't, please revert the whitespace changes.